tests/fixtures/minimal-localizable-web-ext/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "manifest_version": 2,
+  "name": "__MSG_extensionName__",
+  "description": "__MSG_extensionDescription__",
+  "version": "1.0",
+  "applications": {
```

Comment:
You should probably remove `applications` here as well since it's not needed for the test.